products/artifactory.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Artifactory
+category: server-app
+permalink: /Artifactory
+alternate_urls:
+-   /artifactory
+changelogTemplate: https://jfrog.com/help/r/jfrog-release-information/artifactory-__RELEASE_CYCLE__
+iconSlug: jfrog
+releases:
+-   releaseCycle: "7.59"
+    eol: 2024-11-28
+    latest: "7.59.9"
+    latestReleaseDate: 2023-05-28
+
+-   releaseCycle: "7.55"
+    eol: 2024-08-28
+    latest: "7.55.14"
+    latestReleaseDate: 2023-05-23
+
+-   releaseCycle: "7.49"
+    eol: 2024-06-29
+    latest: "7.49.10"
+    latestReleaseDate: 2023-07-03
+
+-   releaseCycle: "7.47"
+    eol: 2024-06-01
+    latest: "7.47.16"
+    latestReleaseDate: 2023-01-30
+
+-   releaseCycle: "7.46"
+    eol: 2024-04-11
+    latest: "7.46.23"
+    latestReleaseDate: 2023-01-31
+
+-   releaseCycle: "7.41"
+    eol: 2024-01-11
+    latest: "7.41.14"
+    latestReleaseDate: 2022-10-11
+
+-   releaseCycle: "7.39"
+    eol: 2023-12-20
+    latest: "7.39.10"
+    latestReleaseDate: 2022-08-03
+
+-   releaseCycle: "7.38"
+    eol: 2023-11-08
+    latest: "7.38.17"
+    latestReleaseDate: 2022-08-11
+
+-   releaseCycle: "7.37"
+    eol: 2023-10-14
+    latest: "7.37.17"
+    latestReleaseDate: 2022-08-04
+
+-   releaseCycle: "7.35"
+    eol: 2023-09-01
+    latest: "7.35.2"
+    latestReleaseDate: 2022-03-09
+
+-   releaseCycle: "7.33"
+    eol: 2023-08-03
+    latest: "7.33.9"
+    latestReleaseDate: 2022-02-07
+
+-   releaseCycle: "7.31"
+    eol: 2023-07-10
+    latest: "7.31.12"
+    latestReleaseDate: 2022-01-25
+
+-   releaseCycle: "7.29"
+    eol: 2024-06-05
+    latest: "7.29.9"
+    latestReleaseDate: 2021-01-11
+
+-   releaseCycle: "7.27"
+    eol: 2023-03-30
+    latest: "7.27.10"
+    latestReleaseDate: 2021-11-07
+
+-   releaseCycle: "7.25"
+    eol: 2023-03-02
+    latest: "7.25.7"
+    latestReleaseDate: 2021-09-10
+
+-   releaseCycle: "6.23"
+    eol: 2022-04-20
+    latest: "6.23.42"
+    latestReleaseDate: 2022-04-14
+
+
+---
+
+> [Artifactory](https://jfrog.com/artifactory/) is a binary repository manager.
+
+JFrog supports all versions of Artifactory from their date of release going forward 18 months. More details can be found on https://jfrog.com/help/r/jfrog-release-information/artifactory-end-of-life
```

Comment:
It would be great to talk about the versioning policy (they seem to follow semantic versioning) and the cloud version.